### PR TITLE
In with the old menu layout

### PR DESCRIPTION
**Pull request by moxiegirl**
We couldn't come to an agreement on how to organize the menus. Since Fred was planning on user testing the result, there was the potential that whatever we came up with tonight, would change again.  So, rather than spin wheels coming up with something that could be tested and subsequently changed, we will go out with the old menu structure. Registry is down in Reference.

See proposed prototype here: http://52.10.125.146:8000/reference/commandline/cli/

Signed-off-by: Mary Anthony mary@docker.com

### CODE REVIEW

**Comment by moxiegirl** [comment]
ping @fredlf @SvenDowideit @jamtur01 @spf13 can I get an LGTM on this new/old layout please :-D

ping @bfirsh @dmp42 @stevvooe this is what we are going out with

File: docs/sources/reference/api/registry_api_client_libraries.md [hunk 1]
```diff
@@ -2,7 +2,7 @@ page_title: Registry API Client Libraries
 page_description: Various client libraries available to use with the Docker registry API
 page_keywords: API, Docker, index, registry, REST, documentation, clients, C#, Erlang, Go, Groovy, Java, JavaScript, Perl, PHP, Python, Ruby, Rust, Scala
 
-# Docker Registry API Client Libraries
+# Docker Registry 1.0 API Client Libraries
```
Comment (SvenDowideit) [inline docs/sources/reference/api/registry_api_client_libraries.md]: so I would have thought this was v1 API clients?

Comment (moxiegirl) [inline docs/sources/reference/api/registry_api_client_libraries.md]: The Registry is at 1.0 I have no idea where these client libraries are at version wise. We don't own those projects.

**Comment by SvenDowideit** [comment]
ok, LGTM - we can worry about correctness later.

**Comment by jamtur01** [comment]
LGTM

**Comment by fredlf** [comment]
Was getting a burrito, but LGTM too!

FWIW, with the new IA, some major sections will have landing/ToC pages so it will be easier to keep links to previous version docs without cluttering up menus.

**Pull request merged**